tests/support/commands.ts

```diff
@@ -75,7 +75,35 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add('openJupyterLab', (): void => {
+Cypress.Commands.add('execDeleteFile', (name: string): void => {
+  cy.exec(`find build/cypress-tests/ -name "${name}" -delete`, {
+    failOnNonZeroExit: false
+  });
+});
+
+Cypress.Commands.add('addFileToPipeline', (name: string): void => {
+  cy.findByRole('listitem', {
+    name: (n, _el) => n.includes(name)
+  }).rightclick();
+  cy.findByRole('menuitem', { name: /add file to pipeline/i }).click();
+});
+
+Cypress.Commands.add('openFile', (name: string): void => {
+  cy.findByRole('listitem', {
+    name: (n, _el) => n.includes(name)
+  }).dblclick();
+});
+
+Cypress.Commands.add('bootstrapFile', (name: string): void => {
+  cy.readFile(`tests/assets/${name}`).then((file: any) => {
+    cy.writeFile(`build/cypress-tests/${name}`, file);
+  });
+});
+
+Cypress.Commands.add('resetJupyterLab', (): void => {
```

Comment:
A lot of the other tests are still using the old name.

Reply:
oh oops, I wonder why linting didn't catch this...

Reply:
oh I guess that would fall under typescript and we don't actually ever compile